api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.module.registrationcore.api.xdssender.impl;
+
+import org.dcm4chee.xds2.common.exception.XDSException;
+import org.openmrs.Patient;
+import org.openmrs.module.registrationcore.api.xdssender.XdsCcdImporter;
+import org.openmrs.module.xdssender.api.domain.Ccd;
+import org.openmrs.module.xdssender.api.model.DocumentInfo;
+import org.openmrs.module.xdssender.api.service.CcdService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.io.IOException;
+
+public class XdsCcdImporterImpl implements XdsCcdImporter {
+
+    @Autowired
+    @Qualifier("ccdService")
```

Comment:
I think that you should use the convention "<artifactId>.<className>"

Reply:
Right, this qualifier has been already defined. Nevertheless, I will correct it.